test/cases/api.sh

```diff
@@ -704,7 +737,7 @@ sudo podman exec osbuild-composer-db psql -U postgres -d osbuildcomposer -c "SEL
 # NOTE: The rest of the job data can contain sensitive information
 #
 # Suppressing shellcheck.  See https://github.com/koalaman/shellcheck/wiki/SC2024#exceptions
-sudo podman exec osbuild-composer-db psql -U postgres -d osbuildcomposer -c "SELECT result->>'Manifest' FROM jobs" | sudo tee "${ARTIFACTS}/manifest.json"
+sudo podman exec osbuild-composer-db psql -U postgres -d osbuildcomposer -c "SELECT result FROM jobs WHERE type='manifest-id-only'" | .data | sudo tee "${ARTIFACTS}/manifest.json"
```

Comment:
This line contains an error `... WHERE type='manifest-id-only'" | .data | sudo ...` producing `/usr/libexec/tests/osbuild-composer/api.sh: line 740: .data: command not found`